agent/agent.go

```diff
@@ -1062,15 +1062,13 @@ func buildCheckRun(ctx context.Context, tc *taskContext) (*apimodels.CheckRunOut
 
 	err = utility.ReadJSONFile(fileName, &checkRunOutput)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "reading checkRun output file")
 	}
 
 	if err := util.ExpandValues(&checkRunOutput, &tc.taskConfig.Expansions); err != nil {
 		return nil, errors.Wrap(err, "applying expansions")
 	}
 
-	tc.logger.Task().Infof("Upserting checkRun: %s.", checkRunOutput.Title)
```

Comment:
Should we maybe keep this but without the checkRunOutput.Title?

Reply:
done

Reply:
looking at this again, we will log `"Successfully upserted checkRun."` later on, so this is not actually necessary. 

The tests might be relying on it though. In that case I'd keep something like "building checkrun" but remove the task id, because it'll be in the task log itself so it's not necessary.

Reply:
removed